doc/Programming-guide/Key-platform-features/Data-collection/File-system-log-appender/index.md

```diff
@@ -19,6 +19,18 @@ You can log in to the Operations service host and browse logs using the `kaa_log
 
 ## Create file system log appender
 
+>**IMPORTANT:** Prior to creating the log appender, you need to specify explicit user privileges in `/etc/sudoers` to allow the user `kaa` to create root path for logs.
+>To do this, run:
+>```
+>sudo visudo
+>```
+>To give full sudo privileges for `kaa` user, add a new line:
+>```
+>kaa ALL=(ALL:ALL) ALL
+>```
+>For more information, see [this thread](http://stackoverflow.com/questions/42290872/file-appender-throws-java-lang-nullpointerexception).
```

Comment:
I propose to remove this line from the documentation. It was mentioned in ticket only for problem description.